src/domain/auth/__tests__/AuthContext.test.tsx

```diff
@@ -39,6 +40,10 @@ jest.mock('oidc-client', () => {
   };
 });
 
+const Wrapper: React.FC<PropsWithChildren> = ({ children }) => {
```

Comment:
`Wrapper` seems a bit too general of a name here - maybe use `NotificationsProviderWrapper` instead?

Reply:
Sure. Changed as suggested